### PR TITLE
Added Recouvrement Des Couts table

### PR DESCRIPTION
**Pull request by dslowikowski**

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by lwyszomi** [comment]
@fyniac

File: custom/intrahealth/report_calcs.py [hunk 1]
```diff
@@ -371,3 +371,66 @@ def total(self, form):
             'date': form.form['mois_visite'],
             'value': value
         }
+
+
+class Recouvrement(fluff.Calculator):
+
+    @fluff.date_emitter
+    def amount_to_pay(self, form):
```
Comment (czue) [inline custom/intrahealth/report_calcs.py]: not a big deal but it looks like all of these could call a shared function with a different property name which would reduce copy/pasted code

File: custom/intrahealth/sqldata.py [hunk 2]
```diff
@@ -612,6 +612,37 @@ def calculate_total_row(self, rows):
...
+        columns.append(DatabaseColumn(_(u"Montant payé"), SumColumn('payments_amount_to_pay')))
+        columns.append(DatabaseColumn(_(u"Payé dans le 30 jours"), SumColumn('payments_in_30_days')))
+        columns.append(DatabaseColumn(_(u"Payé dans le 3 mois"), SumColumn('payments_in_3_months')))
+        columns.append(DatabaseColumn(_(u"Payé dans l`annèe"), SumColumn('payments_in_year')))
+        return columns
+
+    def calculate_total_row(self, rows):
+        total_row = super(RecouvrementDesCouts, self).calculate_total_row(rows)
+        if total_row:
+            total_row[0] = 'Total Region'
```
Comment (czue) [inline custom/intrahealth/sqldata.py]: does this need a translation? or at least an accent?

**Comment by czue** [comment]
will merge but please review comments

**Pull request merged**